### PR TITLE
Add Json for edx.forum.thread.created event.

### PR DESCRIPTION
**Pull request by tehreem-sadat**
**Trello Link:** _https://trello.com/c/gOoQmt4K/11-discussion-forum-event-edxforumthreadcreated_

**Description:** _Users create a new top-level thread, also known as a post, by clicking New Post and then submitting their contributions. When these actions are complete, the server emits an edx.forum.thread.created_

**Checks before merge:**

- [x] Json Reviewed
- [x] Code Reviewed
- [x]  Unit Test Pass
- [x] Commits squashed

### CODE REVIEW

File: openedx/features/caliper_tracking/tests/expected/edx.forum.thread.created.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "action": "Posted",
+  "actor": {
+    "id": "edx.devstack.lms:18000/u/test",
```
Comment (arooshaarif) [inline openedx/features/caliper_tracking/tests/expected/edx.forum.thread.created.json]: Should be replaced with http://localhost:18000

Comment (tehreem-sadat) [reply openedx/features/caliper_tracking/tests/expected/edx.forum.thread.created.json]: done.

Comment (tehreem-sadat) [reply openedx/features/caliper_tracking/tests/expected/edx.forum.thread.created.json]: @arooshaarif kindly review it agian.

**Comment by asamolion** [comment]
@tehreem-sadat Please begin the work on the code.

File: openedx/features/caliper_tracking/tests/current/edx.forum.thread.created.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{
+  "accept_language": "en-US,en;q=0.5",
+  "agent": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:62.0) Gecko/20100101 Firefox/62.0",
+  "context": {
+    "course_user_tags": {},
+    "course_id": "course-v1:edX+DemoX+Demo_Course",
```
Comment (mumarkhan999) [inline openedx/features/caliper_tracking/tests/current/edx.forum.thread.created.json]: Kindly sort that portion.
course_id should come first.

Comment (tehreem-sadat) [reply openedx/features/caliper_tracking/tests/current/edx.forum.thread.created.json]: done

**Pull request merged**